QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -7,15 +7,10 @@
 import ccre.log.Logger;
 
 public class DriveCode {
-    private static FloatStatus leftJoystickChannelX = new FloatStatus();
-    private static FloatStatus leftJoystickChannelY = new FloatStatus();
-    private static FloatStatus rightJoystickChannelX = new FloatStatus();
-    private static FloatStatus rightJoystickChannelY = new FloatStatus();
-
-    public static FloatOutput leftJoystickX = leftJoystickChannelX;
-    public static FloatOutput leftJoystickY = leftJoystickChannelY;
-    public static FloatOutput rightJoystickX = rightJoystickChannelX;
-    public static FloatOutput rightJoystickY = rightJoystickChannelY;
+    public static final FloatStatus leftJoystickX = new FloatStatus();
+    public static final FloatStatus leftJoystickY = new FloatStatus();
+    public static final FloatStatus rightJoystickX = new FloatStatus();
+    public static final FloatStatus rightJoystickY = new FloatStatus();
```

Comment:
The reason for not doing it like this is separation of concerns - not letting users of DriveCode read the values, because these are supposed to be `set`, and allowing reading could be confusing.
However, I'm not convinced that this is critical in this case, so you don't need to revert this change.